pages/summer-of-food-and-fun.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from 'react'
+import { NextPage, GetStaticProps } from 'next'
+
+import { Service, getServices } from '../cms/services'
+import { CardList } from '../src/Components/CardList'
+import { Layout } from '../src/Components/Layout'
+
+interface ListPageProps {
+  services: Array<ServicePreview>
+}
+
+export type ServicePreview = Pick<
```

Comment:
The ServicePreview type is exported from index, so it can be imported from there rather than redefining